roles/confluent.ksql/tasks/rbac.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
This should also add transactionalId role binding:
https://github.com/confluentinc/ksql/pull/4906/files#diff-50a7eb03742964b14b481fd97599fd87L454

Reply:
@stan-confluent Is this the statement we need:

bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --producer --transactional-id ksql-fraud_ --topic _confluent-ksql-fraud__command_topic

Can you confirm why we need it?

Reply:
Based on the link it looks specific to a particular use case, which is not the goal of KSQL installation.  I say this due to the topics being specified for some type of fraud use case.

Reply:
Sorry, I was referring to the line above in that PR (github is not scrolling correctly):
```
The `DESCRIBE` and `WRITE` operations on the `TRANSACTIONAL_ID` with `LITERAL` name `<ksql.service.id>`.
```

Reply:
So this boils down to:
resourceType - TransactionalId
resource - ksql.cluster.id
user - ksql service principal
role - DeveloperWrite (or ResourceOwner)

Reply:
resolving here: https://github.com/domenicbove/cp-ansible/pull/12